kubernetes/client/pod_test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright © 2018 Genome Research Limited
+// Author: Theo Barber-Bany <tb15@sanger.ac.uk>.
+//
+//  This file is part of wr.
+//
+//  wr is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  wr is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with wr. If not, see <http://www.gnu.org/licenses/>.
+
+package client_test
```

Comment:
Is pod tested indirectly by other tests? If so, delete this file. If not, add tests :)

Reply:
Yeah, it's indirectly tested by the scheduler, deployment and e2e tests.